_maps/map_files/cyberiad/z6.dmm

```diff
@@ -8251,7 +8251,7 @@
 /turf/simulated/floor/plasteel,
 /area/derelict/arrival)
 "so" = (
-/obj/structure/chickenstatue{
+/obj/structure/statue/chickenstatue{
 	desc = "Like a MULEbot, but more Russian and less functional.";
 	icon = 'icons/obj/aibots.dmi';
 	icon_state = "mulebot0";
```

Comment:
The mulebot statue should not be a chicken statue thats just resprited and renamed. Please make this an actual subtype.

Reply:
What- that's jank as hell.